concourse/scripts/spec/bosh_pre_destroy.rb

```diff
@@ -0,0 +1,52 @@
+require 'yaml'
```

Comment:
This file should be named ending `_spec.rb` otherwise it won't be picked up and run by default.